internal/driver/config.go

```diff
@@ -30,6 +30,27 @@ type SubscribeInfo struct {
 	Topic        string
 }
 
+type ConnectionInfo struct {
+	Schema   string
+	Host     string
+	Port     string
+	User     string
+	Password string
+	ClientId string
+	Topic    string
+}
+
+const (
```

Comment:
move all const to a separated file, called "protocolpropertykey.go"

Reply:
Modified.